src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -53,4 +63,53 @@
         }
         return new ResponseEntity<>(userService.update(userDto, user), HttpStatus.ACCEPTED);
     }
+
+    @GetMapping("/child")
+    public ResponseEntity<List<ChildDto>> getChildren(Principal principal) {
+        return new ResponseEntity<>(userService.getChildren(getUser(principal.getName())), HttpStatus.OK);
+    }
+
+    @PostMapping("/child")
+    public ResponseEntity<HttpStatus> addChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}") List<@Valid ChildDto> childrenDtoList,
+                                                  Principal principal) {
+        if (childrenDtoList.stream().anyMatch(dto -> dto.getId() > 0)) {
+            throw new IllegalIdentifierException(getExceptionMessageSourceWithAdditionalInfo("exception.illegal.id", "Bigger than zero"));
```

Comment:
It's better to inform "Zero ID is expected" instead of current text and move ```getExceptionMessage...``` method to the next line.